Tables: Visualize tables doc updates (#2100)
wandb/docs: models/tables/visualize-tables.mdx

For information on how to save your current W&B Table view, see [Save your view](#save-your-view).
</Note>

> The following sections describe how to visualize and analyze W&B Tables.

Reviewer: Nit/question: Don't we drop the W&B after first mention?

* **compare the tables at a glance**: apply any operations (sort, filter, group) to both tables in tandem and spot any changes or differences quickly. For example, view the incorrect predictions grouped by guess, the hardest negatives overall, the confidence score distribution by true label, etc.
* **explore two tables independently**: scroll through and focus on the side/rows of interest

> The following section describes two common use cases for comparing W&B Tables: [comparing tables across time](#compare-tables-across-time) and [comparing tables across model variants](#compare-tables-across-model-variants).

Reviewer: ```suggestion
The following section describes how to [compare tables across time](#compare-tables-across-time) and [compare tables across model variants](#compare-tables-across-model-variants).
```
This feels misplaced. I think it should be within the H2 since it is linking into the H3s.

> 

Reviewer: ```suggestion
```

## Compare artifacts

Compare two W&B Tables logged as artifact versions to analyze changes in your data or model performance. Use the merged view or side-by-side view to compare the tables.

### Compare tables across time
Log a table in an artifact for each meaningful step of training to analyze model performance over training time. For example, you could log a table at the end of every validation step, after every 50 epochs of training, or any frequency that makes sense for your pipeline. Use the side-by-side view to visualize changes in model predictions.

<Frame>
    <img src="/images/data_vis/compare_across_time.png" alt="Training progress comparison" max-width="90%"  />
</Frame>

> For a more detailed walkthrough of visualizing predictions across training time, see the [predictions over time report](https://wandb.ai/stacey/mnist-viz/reports/Visualize-Predictions-over-Time--Vmlldzo1OTQxMTk) and this interactive [notebook example](https://colab.research.google.com/github/wandb/examples/blob/master/colabs/datasets-predictions/W%26B_Tables_Quickstart.ipynb?_gl=1*kf20ui*_gcl_au*OTI3ODM1OTcyLjE3MzE0MzU1NjU.*_ga*ODEyMjQ4MjkyLjE3MzE0MzU1NjU.*_ga_JH1SJHJQXJ*MTczMTcwNTMwNS45LjEuMTczMTcwNTM5My4zMy4wLjA.*_ga_GMYDGNGKDT*MTczMTcwNTMwNS44LjEuMTczMTcwNTM5My4wLjAuMA..).

Reviewer: Nit: Cruft in the Colab URL?
